tests/testthat/_snaps/pathogen_summary.md

```diff
@@ -19,10 +19,10 @@
     Output
         pathogen         delay_dist distribution     mean       sd         study
       1 SARS_CoV         incubation        gamma 6.369989 4.085333 Donnelly_etal
-      2 SARS_CoV         incubation        lnorm 4.722382 2.963483  Lessler_etal
-      3 SARS_CoV onset_to_admission        gamma 4.849999 3.491418 Donnelly_etal
+      2 SARS_CoV onset_to_admission        gamma 4.849999 3.491418 Donnelly_etal
+      3 SARS_CoV         incubation        lnorm 4.722382 2.963483  Lessler_etal
```

Comment:
The order should not change here, even if the order in the source data file has changed. We probably need an `order()` call somewhere.

Reply:
Something like this: `params[order(params$pathogen_id), ]`?

Reply:
Should this ordering take place only in the tests or also in the function?

Reply:
Thanks, `order()` ranks capital letters before lowercase and thus the result is not alphabetical order. Should we standardise the use of capitals to solve?